data/json/monsters/zed-classic.json

```diff
@@ -399,6 +399,54 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_swat",
```

Comment:
This should have the kevlar zombie evolution path.

Reply:
The cop zombie doesn't have an evolution path, and the SWAT zombie is not really military (I think? I'm no expert), so I didn't put any evolution for it either.

I thought of maybe having cops transform into SWAT, but that doesn't make sense, as SWAT can be a classic zombie from the start of the Cataclysm.

Are you okay if I hold on for the evolution for now, until we get more feedback about it? :)

Reply:
I'll agree with ymber here with the kevlar evolution path, it makes sense with the swat armor being kevlar based that they integrate it like other zombies, and it will add some more challenge/variety in cities later in the game where the extra challenge is quite needed. The evolution code is also "half-life" based so your swat zombies would never fully disappear either,

Reply:
Fair enough, added upgrade to Kevlar Zombie with same half-life as the soldier zombie.